lib/experimental/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -15,54 +15,7 @@
  *
  * @access private
  */
-class WP_Theme_JSON_Resolver_Gutenberg extends WP_Theme_JSON_Resolver_5_9 {
-
-	/**
-	 * Given a theme.json structure modifies it in place
-	 * to update certain values by its translated strings
-	 * according to the language set by the user.
-	 *
-	 * @param array  $theme_json The theme.json to translate.
-	 * @param string $domain     Optional. Text domain. Unique identifier for retrieving translated strings.
-	 *                           Default 'default'.
-	 * @return array Returns the modified $theme_json_structure.
-	 */
-	protected static function translate( $theme_json, $domain = 'default' ) {
-		if ( null === static::$i18n_schema ) {
-			$i18n_schema         = wp_json_file_decode( __DIR__ . '/theme-i18n.json' );
-			static::$i18n_schema = null === $i18n_schema ? array() : $i18n_schema;
-		}
-
-		return wp_translate_settings_using_i18n_schema( static::$i18n_schema, $theme_json, $domain );
-	}
-
-	/**
-	 * Returns the style variations defined by the theme.
-	 *
-	 * @return array
-	 */
-	public static function get_style_variations() {
-		$variations     = array();
-		$base_directory = get_stylesheet_directory() . '/styles';
-		if ( is_dir( $base_directory ) ) {
-			$nested_files      = new RecursiveIteratorIterator( new RecursiveDirectoryIterator( $base_directory ) );
-			$nested_html_files = iterator_to_array( new RegexIterator( $nested_files, '/^.+\.json$/i', RecursiveRegexIterator::GET_MATCH ) );
-			ksort( $nested_html_files );
-			foreach ( $nested_html_files as $path => $file ) {
-				$decoded_file = wp_json_file_decode( $path, array( 'associative' => true ) );
-				if ( is_array( $decoded_file ) ) {
-					$translated = static::translate( $decoded_file, wp_get_theme()->get( 'TextDomain' ) );
-					$variation  = ( new WP_Theme_JSON_Gutenberg( $translated ) )->get_raw_data();
-					if ( empty( $variation['title'] ) ) {
-						$variation['title'] = basename( $path, '.json' );
-					}
-					$variations[] = $variation;
-				}
-			}
-		}
-		return $variations;
-	}
-
+class WP_Theme_JSON_Resolver_Gutenberg extends WP_Theme_JSON_Resolver_6_0 {
```

Comment:
This file no longer seems to be loaded at all? I'd like to modify `get_theme_data` but I'm not sure how now...

Reply:
Sorry @scruffian, what do you mean? It is listed in `load.php`: https://github.com/WordPress/gutenberg/blob/trunk/lib/load.php#L111.